app/decorators/version_decorator.rb

```diff
@@ -0,0 +1,45 @@
+class VersionDecorator < SimpleDelegator
```

Comment:
I think it would be nice if the name of this decorator said something about why we have chosen to use a decorator rather than simply adding these behaviours on to the version model. If I understand your intentions correctly, it's because we only expect these behaviours to be useful in the context of constructing a document history timeline, but I'm struggling to think of a way to communicate that pithily via a class name